repo_refactoring/code/model.py

```diff
@@ -114,11 +133,30 @@ def write_log(log_file):
 
 class Toy_model(Model):
```

Comment:
do not modify the toy model, you may want to create a new model named as Random_forest_sklearn or something else